nae-spring-core-adapter/src/main/java/com/netgrif/application/engine/adapter/spring/utils/PaginationProperties.java

```diff
@@ -0,0 +1,29 @@
+package com.netgrif.application.engine.adapter.spring.utils;
+
+import lombok.Data;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.context.annotation.Configuration;
+
+/**
+ * Configuration properties class for pagination settings in the application.
+ * This class retrieves and holds pagination-related properties defined under the {@code netgrif.pagination} prefix.
+ */
+@Data
+@Configuration
+@ConfigurationProperties(prefix = "netgrif.pagination")
```

Comment:
There is other PR opened to refactor configuration properties naming scheme. The new naming scheme is the following: `netgrif.<app>.<property-domain>.<property-name>`, so in this case it would be `netgrif.engine.pagination`. Please update it to use new naming convention.